core/src/components/refresher/refresher.ios.vars.scss

```diff
@@ -7,7 +7,7 @@ $refresher-ios-icon-color:            $text-color !default;
 $refresher-ios-text-color:            $text-color !default;
 
 /// @prop - Color of the native refresher spinner
-$refresher-ios-native-spinner-color:  var(--ion-color-step-450, #747577) !default;
+$refresher-ios-native-spinner-color:  var(--ion-color-step-550, var(--ion-text-color-step-450, #747577)) !default;
```

Comment:
This should use a background color token too. We are using the `color` property here, but it's being applied to an SVG which is confusing. Basically the only time we need to use `--ion-text-color-step` is for setting the color of text.

The example in the design document is a bit confusing, so I clarified this on the design doc: https://github.com/ionic-team/ionic-framework-design-documents/pull/238

Also, we don't need to modify the original `--ion-color-step` number.